### ISSUE TITLE
No such file or directory /outputs/compilations

### ISSUE THREAD

**Issue by nonoesp** [brycedrennan/imaginAIry#258]
Hey, Bryce!

I've started testing some of your latest additions; they look great!

After generating images with the `--compilation-anim gif` argument, imaginAIry tries to save images to the `outputs/compilations` directory, but it doesn't exist.

```bash
FileNotFoundError: [Errno 2] No such file or directory: './outputs/compilations'
```

The directory needs to be created beforehand.

**Comment by brycedrennan** [comment]
Oh that's a bug - I'll fix that now.

Let me know if you have any idea on how to make things easier.

**Comment by brycedrennan** [comment]
Fixed in just released 10.0.0
